04_initial_data_prep.R

```diff
@@ -87,7 +87,12 @@ if("ProjectTimeID" %in% colnames(ProjectsInHMIS)){
 
 # This dataset is used when we need an unduplicated concise df for project
 Project0 <<- Project %>% 
```

Comment:
Couldn't make this comment above, so I'm putting it here.

I noticed we're only using ProjectStaging (created on line 26) once throughout the app. Then we immediately use it to create ProjectHMIS. So I think we should combine those two blocks of code, or, if we intended to use ProjectStaging instead of Project thoughout the rest of the app (like when we create Project0), then we need to do a find+replace.